src/Fetcher/FetcherDispatcher.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Fetcher;
+
+use App\Entity\Conference;
+use App\Entity\ConferenceFilter;
+use App\Repository\ConferenceFilterRepository;
+use App\Repository\ConferenceRepository;
+use App\Repository\FetcherConfigurationRepository;
+use Doctrine\ORM\EntityManagerInterface;
+
+class FetcherDispatcher
+{
+    private FetcherConfigurationRepository $fetcherConfigurationRepository;
+    private ConferenceFilterRepository $conferenceFilterRepository;
+    private ConferenceRepository $conferenceRepository;
+    private EntityManagerInterface $em;
+
+    /** @var array<ConferenceFilter> */
+    private array $conferenceFilters;
+    /** @var array<Conference> */
+    private array $existingConferences;
+
+    private int $updatedConferencesCount = 0;
+    private int $newConferencesCount = 0;
+
+    public function __construct(FetcherConfigurationRepository $fetcherConfigurationRepository, ConferenceFilterRepository $conferenceFilterRepository, ConferenceRepository $conferenceRepository, EntityManagerInterface $em)
+    {
+        $this->fetcherConfigurationRepository = $fetcherConfigurationRepository;
+        $this->conferenceFilterRepository = $conferenceFilterRepository;
+        $this->conferenceRepository = $conferenceRepository;
+        $this->em = $em;
+    }
+
+    public function dispatch(FetcherInterface $fetcher): void
```

Comment:
Can you find a better name for this class and method ?

By the way, I expected a class than handle all the fetchers at once, not one by one. I guess you might be confused about the output. You can remove the output from the command. It's a cron anyway.